internal/command/delta.go

```diff
@@ -70,14 +76,24 @@ func delta(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("preparing new deployment: %w", err)
 	}
 
-	// Create a new deployment delta
-	//
-	log.Print("Creating a new deployment delta...\n")
 	client, err := api.NewClient(apiUrl, apiToken, http.DefaultClient)
 	if err != nil {
 		return err
 	}
-	res, err := client.CreateDelta(context.Background(), orgID, appID, delta)
+
+	var res *ht.DeploymentDelta
+	if deltaID == "" {
+
```

Comment:
Please check with the team. I think they do not use extra spacing inside code blocks.

Reply:
(removed)